Support changing temp vol size (#5609)
strimzi/strimzi-kafka-operator: CHANGELOG.md

* Enable Cruise Control anomaly.detection configurations
* Add support for building connector images from the Maven coordinates
* Allow Kafka Connect Build artifacts to be downloaded from insecure servers (#5542)

> * Allow to configure `/tmp` volume size via Pod template. If no template is provided, default value of `1Mi` is used

Reviewer: Maybe this would be better?

```suggestion
* Allow to configure `/tmp` volume size via Pod template. By default `1Mi` is used.
```
